app/code/Bulbulatory/Recommendations/view/adminhtml/ui_component/bulbulatory_recommendations_table_listing.xml

```diff
@@ -108,5 +123,12 @@
                 </item>
             </argument>
         </column>
+        <actionsColumn name="delete" class="Bulbulatory\Recommendations\Ui\Component\Listing\Column\Actions">

```

Comment:
It shouldn't be named `delete` - for now this column contains a delete action only but i can also contain other actions like edit or duplicate or whatever you want so a better name is simply `actions`.

Reply:
@TM18

> It shouldn't be named delete - for now this column contains a delete action only but i can also contain other actions like edit or duplicate or whatever you want so a better name is simply actions.

Changed!